javascript-loops-practice/find-index.js

```diff
@@ -0,0 +1,30 @@
+/* exported findIndex */
+function findIndex(array, value) {
+  var indexOfValue = -1;
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] === value) {
+      indexOfValue = i;
+      break;
+    } else if (array[i] !== value) {
+      continue;
+    }
```

Comment:
You can remove this block. If the above condition isn't met, the loop will continue on its own.